tools/src/main/java/org/apache/kafka/tools/ConnectPluginPath.java

```diff
@@ -172,6 +178,7 @@ private static Set<Path> parseLocations(ArgumentParser parser, Namespace namespa
             }
             String pluginPath = properties.getProperty(WorkerConfig.PLUGIN_PATH_CONFIG);
             if (pluginPath != null) {
+                validatePluginPath(pluginPath, WorkerConfig.PLUGIN_PATH_CONFIG);
```

Comment:
I think we should throw a `ConfigException` when validating `WorkerConfig.PLUGIN_PATH_CONFIG` at this stage. This would preserve the same behavior as when `WorkerConfig.PLUGIN_PATH_CONFIG` is set to an empty value in the config.